bzkanban.js

```diff
@@ -1062,32 +1079,48 @@ function dropCard(ev) {
     }
 }
 
-function dropBacklog(ev) {
-    if (ev.target.classList.contains("drop-target")) {
-        ev.target.classList.remove("drop-target-hover");
-    }
-    ev.preventDefault();
+function showBacklog() {
+    var button = document.getElementById("btnShowBacklog");
+    var backlogCol = document.querySelector("#BACKLOG.board-column");
+
+    if (backlogCol.style.display === "none" || null) {
```

Comment:
And since it's looking like we're going to use that check in two or three places now we should turn that into a function like `isBacklogVisible()`.